src/storage/FirebaseStorageDirective.js

```diff
@@ -0,0 +1,32 @@
+/* istanbul ignore next */
+(function () {
+  "use strict";
+
+  function FirebaseStorageDirective($firebaseStorage, firebase) {
+    return {
+      restrict: 'A',
+      priority: 99, // run after the attributes are interpolated
+      scope: {},
+      link: function (scope, element, attrs) {
+        // $observe is like $watch but it waits for interpolation
+        // Ex: <img firebase-src="{{ myUrl }}"/>
+        attrs.$observe('firebaseSrc', function (newFirebaseSrcVal) {
+          if (newFirebaseSrcVal !== '' &&
```

Comment:
Sorry, I think my comment last time around was confusing. This conditional can be simplified to:

`if (typeof newFirebaseSrcVal === 'string' && newFirebaseSrcVal !== '')`